src/plugins/console/server/lib/proxy_request.ts

```diff
@@ -34,6 +34,16 @@ interface Args {
   rejectUnauthorized?: boolean;
 }
 
+/**
+ * Node http request library does not expect there to be trailing "[" or "]"
```

Comment:
There is a double negation "_does not expect_ ... _but_ ... _does not scrub_ ...". Not sure I fully understand this sentence 😊